src/map/location.hpp

```diff
@@ -110,6 +112,18 @@ struct map_location {
 	map_location rotate_right_around_center(const map_location & center, int k) const;
 
 	friend std::size_t hash_value(map_location const &a);
+
+	int wml_x() const { return x + 1; }
+	int wml_y() const { return y + 1; }
+
+	void set_wml_x(int v) { x = v - 1; }
+	void set_wml_y(int v) { y = v - 1; }
```

Comment:
Well that was my intetion aswell but it not that easy, in any case i think this commit makes it easier to port map_location to use the wml format later if we really want to do that.